backend/collaboration-service/app/src/routes/room.ts

```diff
@@ -20,11 +20,11 @@ roomRouter.post("/", async (req: Request<{}, {}, { difficulties: string[]; topic
 
   const { difficulties, topics } = req.body;
 
-  if (!difficulties || !(difficulties.every((d: string) => Object.keys(DIFFICULTY).includes(d)))) {
+  if (!difficulties || !(difficulties.every((d: string) => Object.values(DIFFICULTY).includes(d as DifficultyValue)))) {
```

Comment:
i think users should be able to indicate no difficulty/no topic if they are okay with any difficulty/topic

Reply:
faede7c8b0d044f8c2c302f61423fcd6ab121953